editors/code/src/config.ts

```diff
@@ -86,58 +88,91 @@ export class Config {
      * [1]: https://github.com/Microsoft/vscode/issues/11514#issuecomment-244707076
      */
     private configureLanguage() {
-        if (this.typingContinueCommentsOnNewline && !this.configureLang) {
+        // Only need to dispose of the config if there's a change
+        if (
+            this.configureLang &&
+            this.typingContinueCommentsOnNewline !==
+                this.configureLang.typingContinueCommentsOnNewline
```

Comment:
I don't think we need this check, it's fine to reload the on enter stuff whenever this is called now

Reply:
Removed :+1: